datasets/un_ga/README.md

```diff
@@ -0,0 +1,142 @@
+---
+annotations_creators:
+- found
+language_creators:
+- found
+languages:
+- ar
+- en
+- es
+- fr
+- ru
+- zh
```

Comment:
The languages depend on the config name. Could you format the languages accordingly ?
For example
```
languages:
  ar-to-es:
  - ar
  -es
```

Reply:
@lhoestq Added language tags and BibTex :smile: